internal/core/items/path.rs

```diff
@@ -148,7 +148,7 @@ impl Path {
         let viewbox_width = self.viewbox_width();
         let viewbox_height = self.viewbox_height();
 
-        let mut elements_iter = self.elements().iter();
+        let mut elements_iter = self.elements().iter()?;
```

Comment:
Maybe move that up in the function so we don't query all the other property in case the path is empty

Reply:
Good idea